cmd/list-role-eligibility-schedule-instance.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2025 Specter Ops, Inc.
+//
+// This file is part of AzureHound.
+//
+// AzureHound is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// AzureHound is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"context"
+	"github.com/bloodhoundad/azurehound/v2/client"
+	"github.com/bloodhoundad/azurehound/v2/client/query"
+	"github.com/bloodhoundad/azurehound/v2/enums"
+	"github.com/bloodhoundad/azurehound/v2/models"
+	"github.com/bloodhoundad/azurehound/v2/panicrecovery"
+	"github.com/bloodhoundad/azurehound/v2/pipeline"
+	"github.com/spf13/cobra"
+	"os"
+	"os/signal"
+	"time"
+)
+
+func init() {
+	listRootCmd.AddCommand(listUnifiedRoleEligibilityScheduleInstanceCmd)
+}
+
+var listUnifiedRoleEligibilityScheduleInstanceCmd = &cobra.Command{
+	Use:          "unified-role-eligibility-schedule-instances",
+	Long:         "Lists Unified Role Eligibility Schedule Instances",
+	SilenceUsage: true,
+	Run:          listUnifiedRoleEligibilityScheduleInstancesCmdImpl,
+}
+
+func listUnifiedRoleEligibilityScheduleInstancesCmdImpl(cmd *cobra.Command, args []string) {
+	ctx, stop := signal.NotifyContext(cmd.Context(), os.Interrupt, os.Kill)
+	defer gracefulShutdown(stop)
+
+	log.V(1).Info("testing connections")
```

Comment:
With the `connectAndCreateClient()` function, we already output the "testing connections" log line, it can be removed here